.gitignore

```diff
@@ -0,0 +1 @@
+*.pyc
```

Comment:
great work here, I can tell you'll make a great programmer someday

Reply:
🤩 You as well, amazing review!